Amplify/Core/Configuration/AmplifyConfiguration.swift

```diff
@@ -137,7 +137,8 @@ extension Amplify {
 
         // Always configure Logging, Hub and Auth first, so they are available to other categories.
         // Auth is a special case for other plugins which depend on using Auth when being configured themselves.
-        let manuallyConfiguredCategories = [CategoryType.logging, .hub, .auth]
+        try configure(CategoryType.logging.category, using: resolvedConfiguration)
```

Comment:
Isn't this the same as before? 🤔

Reply:
Logging needs to be the first, order wasn't guaranteed in the previous code and initialization would randomly fail..  I've added additional code comments to clarify.

Reply:
> order wasn't guaranteed

in arrays!? TIL 🤯

Reply:
Could it be that there's something else going on behind the scenes that's causing the fail? Arrays should keep their order, it's like one of their features.

It could totally be that Swift's for loop is messing up, I'm just being extra cautious